preact.config.js

```diff
@@ -33,28 +38,22 @@ export default function (config, env, helpers) {
 		/babel-standalone/
 	].concat(config.module.noParse || []);
 
-	const babel = helpers.getLoadersByName(config, 'babel-loader')[0].rule;
+	const { rule: babel } = helpers.getLoadersByName(config, 'babel-loader')[0];
 	babel.exclude = [/babel-standalone/].concat(babel.exclude || []);
 
 	// Add CSS Custom Property fallback
-	const cssConfig = config.module.rules.filter(d => d.test.test('foo.less'));
-	cssConfig.forEach(c => {
-		c.use.filter(d => d.loader == 'postcss-loader').forEach(x => {
-			x.options.plugins.push(customProperties({ preserve: true }));
-		});
-	});
+	const { loader: postcssLoader } = helpers.getLoadersByName(config, 'postcss-loader')[0];
+	postcssLoader.options.postcssOptions.plugins.push(customProperties({ preserve: true }));
 
 	// Fix keyframes being minified to colliding names when using lazy-loaded CSS chunks
-	const optimizeCss = config.optimization && (config.optimization.minimizer || []).filter(plugin => /^OptimizeCssAssets(Webpack)?Plugin$/.test(plugin.constructor.name))[0];
-	if (optimizeCss) {
+	if (!env.isServer) {
+		const optimizeCss = config.optimization.minimizer.find(plugin => plugin.constructor.name == 'OptimizeCssAssetsWebpackPlugin');
 		optimizeCss.options.cssProcessorOptions.reduceIdents = false;
 	}
```

Comment:
This is slightly more fragile now, but that's the intention; we want this to break if `optimize-css-assets-webpack-plugin` cannot be found.